anomaly_detection/api/middleware/auth.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from werkzeug.wrappers.request import Request
+
```

Comment:
project_id should be changed to tenant_id

Reply:
done